react-blocks/src/blocks/Submenu/Submenu.js

```diff
@@ -105,7 +105,7 @@ export class Submenu extends Component {
             attributes={{
               submenu_style: this.props.submenu_style,
               title: this.props.title,
-              levels: this.props.levels,
+              levels: this.props.levels
```

Comment:
Oops, this wasn't intended, the trailing comma is supposed to be better for version control, but I just hate seeing it!